README.md

```diff
@@ -41,7 +41,22 @@ in your commit messages so it is clear what each commit is for._" <br />
 when you want someone to review your work..._" <br />
 > "_Please only assign a Pull Request for review
 once all the tests have passed on CI..._" <br />
-> For a **list** of these common scenarios see:
+
+We've devised a set of rules and actions to respond to these problems, they look like this:
+
+| Category      | Rule                                                                               | dwylbot comments                                                                                                                                                                                                                                                                                                      | Labels changed                                                                   |
```

Comment:
Right, that's interesting. So in theory, say I created an issue with a blank description and dwylbot warned me against this. I then continue working on the issue and adding labels to it/ closing it etc. - I should only receive the warning from dwylbot that I should give my issue a description once, when I first created it? I won't get a reminder every time I do another action on the issue?